blockchain/validators/src/test_utils.rs

```diff
@@ -26,3 +26,131 @@ pub fn make_metadata(seed: u64) -> BlockMetadata {
     )
     .expect("BlockMetadata::from_contents_and_keypair")
 }
+
+use crate::metadata::avr::{AvrHistoryConfig, AvrHistoryRecord};
+use mc_common::ResponderId;
+use std::str::FromStr;
+
+/// Get first two actual Intel Attestation Verification Reports for
+/// node1.prod.mobilecoinww.com in order to build control data
+/// for tests
+pub fn get_ias_reports() -> (VerificationReport, VerificationReport) {
```

Comment:
does this need to be `pub`?
```suggestion
fn get_ias_reports() -> (VerificationReport, VerificationReport) {
```